pkg/gen/input/workflows/internal/file/create_release_branch.go

```diff
@@ -0,0 +1,136 @@
+package file
+
+import (
+	"github.com/giantswarm/devctl/pkg/gen/input"
+	"github.com/giantswarm/devctl/pkg/gen/input/workflows/internal/params"
+)
+
+func NewCreateReleaseBranchInput(p params.Params) input.Input {
+	i := input.Input{
+		Path:         params.RegenerableFileName(p, "create_release_branch.yaml"),
+		TemplateBody: createReleaseBranchTemplate,
+	}
+
+	return i
+}
+
+var createReleaseBranchTemplate = `# DO NOT EDIT. Generated with:
+#
+#    devctl gen workflows
+#
+
+# Creates a branch for the 'previous' minor version when a 'new' minor version is tagged
+
+name: create-minor-version-branch
+
+on:
+  push:
+    tags: ['v*.*.*']
+
+jobs:
```

Comment:
I'd like to add that to every workflow we have here because it makes further debugging and improvements easier.

Could you also show me the output of that job for and example workflow run? I have a hunch we can simplify the code from `create-release-branch` a lot.

```suggestion
jobs:
  debug_info:
    name: Debug info
    runs-on: ubuntu-18.04
    steps:
      - name: Print github context JSON
        run: |
          cat <<EOF
          ${{ toJson(github) }}
          EOF
```

Reply:
Sure, what are you looking for? Here's the likely relevant part, but if you're hoping GH provides the previous tag or something, I didn't find one.

```
{
  "token": "***",
  "job": "debug_info",
  "ref": "refs/tags/v3.5.0",
  "sha": "397fb75698f44ea5f3f289f17e860204da89e8c3",
  "repository": "stone-z/create-minor-branch-action",
  "repository_owner": "stone-z",
  "repositoryUrl": "git://github.com/stone-z/create-minor-branch-action.git",
  "run_id": "156462941",
  "run_number": "39",
  "actor": "stone-z",
  "workflow": "create-minor-version-branch",
  "head_ref": "",
  "base_ref": "",
  "event_name": "push",
  "event": {
    "after": "397fb75698f44ea5f3f289f17e860204da89e8c3",
    "base_ref": "refs/heads/master",
    "before": "0000000000000000000000000000000000000000",
    "commits": [],
    "compare": "https://github.com/stone-z/create-minor-branch-action/compare/v3.5.0",
    "created": true,
    "deleted": false,
    "forced": false,
    "head_commit": {
      "author": {
        <redacted, but it's me>
      },
      "committer": {
        <redacted, but it's me>
      },
      "distinct": true,
      "id": "397fb75698f44ea5f3f289f17e860204da89e8c3",
      "message": "Original with refactor",
      "timestamp": "2020-07-03T12:26:54+02:00",
      "tree_id": "eca42b58132a65e7f095ab530d0521a9760b8e9e",
      "url": "https://github.com/stone-z/create-minor-branch-action/commit/397fb75698f44ea5f3f289f17e860204da89e8c3"
    },
    "ref": "refs/tags/v3.5.0",
    "repository": {
      "created_at": 1592916067,
      "default_branch": "master",
	... a bunch of junk ...
```

Reply:
Added the debug info job to the workflow, but I guess this will already be present in `Create Release` anyway